.github/workflows/python-package.yml

```diff
@@ -14,12 +14,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8]
+        python-version: [3.11]
```

Comment:
```suggestion
        python-version: ["3.11"]
```

Reply:
(and throughout; otherwise when we eventually reach `3.20`, yaml will parse it as `3.2`!)

Reply:
That's true, 3.20 is the same as 3.2, i.e. must use "3.20".
BTW, this is valid for "3.10" - I skipped it, but it could soon become the "legacy" version.

So, quotes are mandatory for versions that ends in zeros, otherwise - recommended. I suggest quoting all the versions. Is this ok?